Makefile

```diff
@@ -234,8 +235,17 @@ worker-key-pair: ca
 	rm /etc/osbuild-composer/worker-csr.pem
 
 .PHONY: unit-tests
+.ONESHELL:
 unit-tests:
-	go test -race ./...
+	go test -race -covermode=atomic -coverprofile=coverage.txt -coverpkg=$$(go list ./... | grep -v rpmmd/test$ | tr "\n" ",") ./...
+	# go modules with go.mod in subdirs are not tested automatically
+	cd pkg/splunk_logger
+	go test -race -covermode=atomic -coverprofile=../../coverage_splunk_logger.txt -coverpkg=$$(go list ./... | grep -v rpmmd/test$ | tr "\n" ",") ./...
```

Comment:
The filtering out of rpmmd/test (`go list ./... | grep -v rpmmd/test$`) is a remnant from a long long time ago.  There is no package matching `rpmmd/test` anymore:
```
› go list ./... | grep -F rpmmd/test -c
0
```

Good opportunity for a clean up, I think.

Reply:
the whole line will be removed by https://github.com/osbuild/splunk_logger/issues/1 soonish
but I can fix it for the line 240 here ;)